### PR TITLE
fix: add missing PermissionResource types from Cloud specifications

### PR DESCRIPTION
**Pull request by bednar**
## Proposed Changes

Added missing `Resource` types.

## Checklist

- [x] CHANGELOG.md updated
- [x] Rebased/mergeable
- [x] `yarn test` completes successfully
- [x] Commit messages are in [semantic format](https://seesparkbox.com/foundry/semantic_commit_messages)
- [x] Sign [CLA](https://www.influxdata.com/legal/cla/) (if not already signed)

### CODE REVIEW

File: packages/apis/src/generated/types.ts [hunk 1]
```diff
@@ -2581,6 +2581,8 @@ export interface Resource {
     | 'annotations'
     | 'remotes'
     | 'replications'
+    | 'flows'
```
Comment (sranka) [inline packages/apis/src/generated/types.ts:2584]: Is this really generated? ... I guess it is not, so the next generation of APIs can yield to unpredictable results

Comment (bednar) [reply packages/apis/src/generated/types.ts:2584]: It is generated by our fork of `oats`, see https://github.com/bonitoo-io/oats/pull/2

Comment (sranka) [reply packages/apis/src/generated/types.ts:2584]: OK, .... ready to go once a solution is in `bonitoo-io/oats` master branch.

**Comment by bednar** [comment]
@sranka switched implementation to use `--patchScript` option from `bonitoo-io/oats`

**Review by sranka — APPROVED** [review]
:+1:

**Pull request merged**